src/tools/etherscanClient.ts

```diff
@@ -179,6 +180,7 @@ const convertEtherScanTxToTransactionData = (
     timestamp: Number(txData.timeStamp),
     to: txData.to === "" ? null : txData.to.toLowerCase(),
     value: txData.value,
+    input: txData.input,
     truncatedInput: txData.input.slice(0, 10),
```

Comment:
Instead of `input`, call this field `inputData` to be more specific.
Also we can remove the `truncatedInput` now.